modules/capacity_factors_wind_pv/workflow/internal/internal_config.yaml

```diff
@@ -0,0 +1,3 @@
+# This file contains configuration that users should not modify.
+resources:
+  shapefile_url:
```

Comment:
Looking at `config/default.yaml`, I think there are a bunch of things you want to pass here to protect them from user error (and to avoid having to define a `schema` for them).

Good candidates:
- Downloaded datasources (eez, cap-fact).
- The spatial scope (unless those datasources are global, in which case it makes no sense to have a spatial scope).
- The min-max capacity range.
- The spatial 'bounds' (again, if the datasource is global, this could be removed).
- The thresholds

Reply:
Done.